src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/recordings/test_containerappjob_update_min_max_executions_zero_e2e.yaml

```diff
@@ -0,0 +1,6543 @@
+    headers:
+      cache-control:
+      - no-cache
+      content-length:
+      - '35146'
+      content-type:
+      - application/json; charset=utf-8
+      date:
+      - Sat, 02 Aug 2025 09:51:08 GMT
+      expires:
+      - '-1'
+      pragma:
+      - no-cache
+      strict-transport-security:
+      - max-age=31536000; includeSubDomains
+      x-cache:
+      - CONFIG_NOCACHE
+      x-content-type-options:
+      - nosniff
+      x-ms-ratelimit-remaining-subscription-global-reads:
+      - '16499'
+      x-msedge-ref:
+      - 'Ref A: 14D13BB8344D4C688CC933C5069D7D94 Ref B: PNQ231110909036 Ref C: 2025-08-02T09:51:09Z'
+    status:
+      code: 200
+      message: OK
+- request:
+    body: '{"location": "East US", "identity": {"type": "None", "userAssignedIdentities":
+      null}, "properties": {"environmentId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/client.env_rg_eastus/providers/Microsoft.App/managedEnvironments/env-eastus",
+      "configuration": {"secrets": null, "triggerType": "event", "replicaTimeout":
+      200, "replicaRetryLimit": 2, "manualTriggerConfig": null, "scheduleTriggerConfig":
+      null, "eventTriggerConfig": {"replicaCompletionCount": 1, "parallelism": 1,
+      "scale": {"minExecutions": 1, "maxExecutions": 5, "pollingInterval": 30, "rules":
+      [{"name": "azure-servicebus", "azureQueue": null, "custom": null, "http": null,
+      "type": "azure-servicebus", "metadata": {"topicName": "my-topic", "messageCount":
+      "5"}, "auth": [{"triggerParameter": "secretRef", "secretRef": "connection-string-secret"}]}]}},
+      "registries": null, "dapr": null}, "template": {"containers": [{"image": "mcr.microsoft.com/k8se/quickstart-jobs:latest",
+      "name": "eventjob000002", "command": null, "args": null, "env": null, "resources":
+      {"cpu": 0.25, "memory": "0.5Gi"}, "volumeMounts": null}], "initContainers":
+      null, "volumes": null}, "workloadProfileName": null}, "tags": null}'
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate
+      CommandName:
+      - containerapp job create
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '1188'
+      Content-Type:
+      - application/json
+      ParameterSetName:
+      - --resource-group --name --environment --replica-timeout --replica-retry-limit
+        --trigger-type --parallelism --replica-completion-count --min-executions --max-executions
+        --polling-interval --scale-rule-name --scale-rule-type --scale-rule-metadata
+        --scale-rule-auth --image --cpu --memory
+      User-Agent:
+      - python/3.12.3 (Linux-6.6.87.1-microsoft-standard-WSL2-x86_64-with-glibc2.39)
+        AZURECLI/2.76.0
+    method: PUT
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.App/jobs/eventjob000002?api-version=2025-02-02-preview
```

Comment:
I notice in the azure-cli recording file the `api-version` is `2025-02-02-preview`:
uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.App/jobs/eventjob000002?api-version=2025-02-02-preview

Due to in the CI pipeline, it will not install containerapp extension, so the api-version should be `2025-03-01`.
You can see the integration test CI failed with api-version mismatch.
To fix this, you need to uninstall extension first, then run this test again:
```
az extension remove -n containerapp
azdev test run containertapp.test_containerappjob_update_min_max_executions_zero_e2e
```